packages/ui/src/common/providers/network-endpoints/provider.tsx

```diff
@@ -35,6 +35,7 @@ export const NetworkEndpointsProvider = ({ children }: Props) => {
         queryNodeEndpoint: config['graphql_server'],
         membershipFaucetEndpoint: config['member_faucet'],
         nodeRpcEndpoint: config['websocket_rpc'],
+        configEndpoint,
```

Comment:
```suggestion
        configEndpoint: config['config'],
```
It's available on: https://54.163.55.37.nip.io/network/config.json